packages/hoppscotch-cli/src/utils/mutators.ts

```diff
@@ -39,49 +34,46 @@ export const parseErrorMessage = (e: unknown) => {
   return msg.replace(/\n+$|\s{2,}/g, "").trim();
 };
 
+export async function readJsonFile(path: string) {
+  if(!path.endsWith('.json')) {
+    throw error({ code: "INVALID_FILE_TYPE", data: path })
+  }
+  try {
+    await fs.access(path)
+  } catch (e) {
+    throw error({ code: "FILE_NOT_FOUND", path: path })
+  }
+  try { 
+    return JSON.parse((await fs.readFile(path)).toString())
+  } catch(e) {
+    throw error({ code: "UNKNOWN_ERROR", data: e })
+  }
```

Comment:
It is good to have an empty line between sections of code that do a specific thing. Gives the impression of an isolation and gives the code a bit of 'room to breathe' (readability).

```suggestion
  if(!path.endsWith('.json')) {
    throw error({ code: "INVALID_FILE_TYPE", data: path })
  }
  
  try {
    await fs.access(path)
  } catch (e) {
    throw error({ code: "FILE_NOT_FOUND", path: path })
  }
  
  try { 
    return JSON.parse((await fs.readFile(path)).toString())
  } catch(e) {
    throw error({ code: "UNKNOWN_ERROR", data: e })
  }
```

Reply:
Spaced out at https://github.com/hoppscotch/hoppscotch/pull/2907/commits/e94fb04215071caf6ab5b1910d6df0a6a63294a1#diff-fccb5e9c3967b37fa6f0242a9bdddcfcad1f7e7baf5f4508f95730ccf78bdf97R41